src/controllers/TalksController.php

```diff
@@ -289,6 +367,12 @@ public function createTalkAction(Request $request, PDO $db)
         $tz = new DateTimeZone($event['tz_continent'] . '/' . $event['tz_place']);
         $talk['date'] = (new DateTime($start_date, $tz))->format('U');
 
+        $event_start_date = (new DateTime($event['start_date']))->format('U');
```

Comment:
This uses the servers timezone. Is that wise? Or shouldn't we include the timezone-information here as well?